### DISCUSSION TITLE
Home (x and y) Axis Offsets, how should it work?

### DISCUSSION THREAD

**Discussion by Friday13th87** [mriscoc/Ender3V2S1#489] [Q&A]
Hello there,
i know that there is a Wiki page [https://github.com/mriscoc/Ender3V2S1/wiki/How-to-set-the-home-offsets] about the home offsets.

In the Wiki page it says, that the correct way is to set the x/y_min/max positions. So far, so good.

But if i set those values to my printer setup to x_min=-8 and y_min=-11 that doesn not mean that the actual values for Position:0/0/z are changing.
What i mean: my printing area is 230x230 with the values:
X_MIN_POS = -8;  [---> this should be my new x=0, but it is not]
X_MAX_POS = 222; [---> this should be my new x=230, but it is not]
Y_MIN_POS = -11; [---> this should be my new y=0, but it is not]
Y_MAX_POS = 219; [---> this should be my new y=230, but it is not]

But if i send my printer to the position 0/0/z it still goes to 0/0/z  and that 0/0/z is not the physical -8/11/z, so i can move the nozzle to those values but i cant actually print on them through my slicer as the slicer is not using the minus values nor dies the firmware is changing those values (so if x=5 and y=5 is coming from the gcode file, it should actually be x=-3 and y=-6)

In my previous firmwares i was able to put the home offset values and so my 0/0/z position changed to my bed minimum position.

How do i get my printer to print on x/y=-8/-11?

**Comment by mriscoc** [comment]
Check: https://github.com/mriscoc/Ender3V2S1/wiki/Calibration-Guides
(0,0) must be the front left corner of your bed.
